vars/transportRequestUploadFile.groovy

```diff
@@ -257,19 +257,18 @@ void call(Map parameters = [:]) {
 
                         echo "[INFO] Uploading application '${configuration.applicationName}' to transport request '${configuration.transportRequestId}'."
 
-                        cm.uploadFileToTransportRequestCTS(
```

Comment:
Would this call groovy code?

Reply:
Yes. `uploadFileToTransportRequestCTS` is a method of the `com.sap.piper.cm.ChangeManagement `class which implements the  functionality by Groovy.